### ISSUE TITLE
lazy pricks why don't you just build it yourself i have to download the vs studio 15 gb to run this shitty script

### ISSUE THREAD

**Issue by akmlvl** [robertdavidgraham/masscan#359]

**Comment by badgenes** [comment]
and once you have studio installed, wrap this awesome code RDG has shared with your own code to do all of the things you want rather than always ask.  you will learn quite a lot in the process and end up with your own tools that leverage masscan.  i admire rdg, i'd share mine but too many concerns.  if i'd done anything like that, I mean

**Comment by kruglinski** [comment]
You are definitely windows guy, if you work in linux only the builtin toolchan is all you need

**Comment by badgenes** [comment]
Yes, actually use them all but make money from Windows among other things.
Linux I believed would take over the world in the 90s...after that didn't
happen I stuck with what paid the bills for me. But OSX is where it's at
for infosec imho. Xojo for windows makes it easy to Port to osx, take some
laser focused highly agile vb.net / SQL app feeding masscan the coordinates
with final handoff to nmap... beautiful stuff
